docs/content/docs/custom-resource/pod-template.md

```diff
@@ -100,6 +100,66 @@ spec:
     parallelism: 2
 ```
 
+We also provide a way to obtain the jar package remotely, which can avoid you repeating the packaging image every time
```

Comment:
nit: "package remotely, which can avoid you repeating the packaging image every time" -> "package remotely to avoid repeating the image packaging on each iteration."